aiida_quantumespresso/workflows/ph/base.py

```diff
@@ -117,8 +135,8 @@ def handle_out_of_walltime(self, node):
         return ProcessHandlerReport(True)
 
     @process_handler(priority=410, exit_codes=PhCalculation.exit_codes.ERROR_CONVERGENCE_NOT_REACHED)
-    def handle_convergence_not_achieved(self, node):
-        """Handle `ERROR_CONVERGENCE_NOT_REACHED` exit code: decrease the mixing beta and restart from scratch."""
+    def handle_convergence_not_reached(self, node):
```

Comment:
Why change this method name?

Reply:
In fact, there are two changes here :smile: 
First I change the function name from `achieved` to `reached`, I guess this should be the same as the exit status name? Any reason they are not the same previously?
Second, I remove the `from scratch` from the docstring since I think the setting here use the `recover=True` and it is not run from scratch. But as you suggest below, we need to ask someone more familiar with ph.x.

Reply:
> First I change the function name from achieved to reached, I guess this should be the same as the exit status name? Any reason they are not the same previously?

Fair enough, it is just that other base workchain (`PwBaseWorkChain` for example) have the same handler. If we make this change, would be good to do it there as well for consistency.

And I noticed the change in the docstring and didn't comment because I agree with it :)